xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.h

```diff
@@ -172,5 +173,8 @@ class CDVDInputStreamBluray
 #endif
 
   private:
+    bool OpenStream(CFileItem &item);
     void SetupPlayerSettings();
+    std::unique_ptr<CDVDInputStreamFile> m_pstream;
+    std::unique_ptr<std::string> m_rootPath;
```

Comment:
this could just be a regular `std::string` and pass it around using & as I understand the code. Simplifies things a bit.